cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1077,11 +1077,11 @@ String getAncillaryConfigMapKeyLogConfig() {
         return ANCILLARY_CM_KEY_LOG_CONFIG;
     }
 
-    public static String getClusterCaName(String cluster)  {
+    public static String clusterCaSecretName(String cluster)  {
```

Comment:
Could we make this `clusterCaCertSecretName()`? It's just a bit more explicit that the `Secret` concerned is the one containing the certificate.

Reply:
Agree, I'll do the same for `clientsCaSecretName` in `KafkaCluster` renaming to `clientsCaCertSecretName`